evalml/tests/model_understanding_tests/test_partial_dependence.py

```diff
@@ -809,7 +809,7 @@ def test_graph_partial_dependence_regression_and_binary_categorical(
     )
     plot_data = fig.to_dict()["data"][0]
     assert plot_data["type"] == "bar"
-    assert plot_data["x"] == ["0", "1", "2"]
+    assert all(plot_data["x"] == ["0", "1", "2"])
```

Comment:
This was throwing an error as now the new behavior when comparing a numpy array and a list is to return an iterable with elementwise comparison, rather than an implicit all().  Now the all() is explicit and Gucci reigns again.